gulpfile.js

```diff
@@ -0,0 +1,80 @@
+var args            = require('yargs').argv;
+var bump            = require('gulp-bump');
+var checktextdomain = require('gulp-checktextdomain');
+var gulp            = require('gulp');
+var potomo          = require('gulp-potomo');
+var pseudo          = require('gulp-pseudo-i18n');
+var rename          = require('gulp-rename');
+var runSequence     = require('run-sequence');
+var sort            = require('gulp-sort');
+var wpPot           = require('gulp-wp-pot');
+var yaml            = require('yamljs');
+
+var config = yaml.load('config.yml');
+
+/* Language Tasks
+ -------------------------------------------------- */
+gulp.task('languages', function() {
+  return runSequence('po', 'mo')
+});
+
```

Comment:
This gulp task automatically generates po/mo language files and alerts you to any translation strings missing the plugin text-domain.
`gulp languages`